source/_components/android_ip_webcam.markdown

```diff
@@ -25,17 +25,46 @@ android_ip_webcam:
   - host: 192.168.1.10
 ```
 
-Configuration variables:
-
-- **host** (*Required*): The IP Address of the phone on the network.
-- **port** (*Optional*): Default is set 8080. The port the IP Webcam listens on.
-- **name** (*Optional*): Override the name of the phone.
-- **username** (*Optional*): The username to access the phone.
-- **password** (*Optional*): The password to access the phone.
-- **scan_interval** (*Optional*): Default is 10 seconds. Defines the update interval of the phone.
-- **sensors** array (*Optional*): Conditions to display sensor in the frontend. See the list of supported sensors.
-- **switches** array (*Optional*): Conditions to display settings in the frontend. See the list of supported settings.
-- **motion_sensor** (*Optional*): Activate motion sensor if auto_discovery is disabled.
+{% configuration %}
+host:
+  description: The IP Address of the phone on the network.
+  required: true
+  type: string
+port:
+  description: The port the IP Webcam listens on.
+  required: false
+  default: 8080
+  type: integer
+name:
+  description: Override the name of the phone.
+  required: false
+  type: string
+username:
+  description: The username to access the phone.
+  required: false
+  type: string
+password:
+  description: The password to access the phone.
+  required: false
+  type: string
+scan_interval:
+  description: Defines the update interval of the phone.
+  required: false
+  default: 10
+  type: integer
+sensors:
+  description: Conditions to display sensor in the frontend. See the list of supported sensors.
+  required: false
+  type: list
+switches:
+  description: Conditions to display settings in the frontend. See the list of supported settings.
+  required: false
+  type: list
```

Comment:
A list of `keys` is expected here, which is now listed in a lower part of the documentation.